libs/util/shared/src/lib/models/search.model.ts

```diff
@@ -33,6 +33,17 @@ export interface MetadataLink {
   url: string
 }
 
+// we will need to change model to contain format:
+// export type MetadataLinkFormats = 'OGC:WMS'|'OGC:WFS'|'OGC:WPS'|'ESRIFeature'|'csv'|'geosjon'|'json'|'shp'|'xls'|'xlsx'|'kml'|'gpkg'|'outbound'|'unknown'
+
+// export interface MetadataLink {
```

Comment:
why ?

Reply:
this is something that we discussed, this model would be more appropriate for all kind of links, be it OGC services or simple files; the concept of "protocol" does not make sense for files, and the model currently does not contain a `title` (only a `name` which is more of a technical identifier)